rfc/text/advanced/ap_10_rpc_progressive_call_invocations.md

```diff
@@ -126,19 +126,19 @@ To bind `INVOCATION` messages to the same *Callee*, the *Caller* can specify the
         CALL.Options.sticky|bool := true
 
 
-In this case, the *Dealer* must make a first `INVOCATION` based on `<invocation_policy>` and then route all 
+In this case, the *Dealer* must make a first `INVOCATION` based on `<invocation_policy>` and then route all
 subsequent `progressive` calls to the same *Callee*.
 
-If binding all ongoing `progressive` calls to the same *Callee* is not required, the *Caller* can set the `sticky` option to `false`. 
+If binding all ongoing `progressive` calls to the same *Callee* is not required, the *Caller* can set the `sticky` option to `false`.
```

Comment:
Hmmm this part was removed right? Or is it because this change is based on an older version of the docs?

Reply:
kinda Yes and No :)
The original PR with removal was merged but then due to conflicts was reverted. So we have an opened #460 once again. But it needs to be merged after this one. And then — everything should be all good %)

Reply:
FYI @Jopie64 Merged PR with sticky :)